gridstatus/gs_logging.py

```diff
@@ -1,3 +1,24 @@
+import logging
+
+
+def setup_gs_etl_logger():
+    logger = logging.getLogger("gs_etl")
+    logger.setLevel(logging.INFO)  # Set the logger level to DEBUG
+    handler = logging.StreamHandler()
+    handler.setLevel(logging.INFO)
+    handler.setFormatter(
+        logging.Formatter(
+            "%(asctime)s - %(levelname)s - %(message)s",
+            datefmt="%Y-%m-%d %H:%M:%S",
+        ),
+    )
+    logger.addHandler(handler)
+    return logger
+
+
+logger = setup_gs_etl_logger()
+
+
 def log(msg, verbose=False):
```

Comment:
Definitely can, though it might be a separate PR. There are about 200+ places where the existing `log` is called, mostly with two arguments `(msg, verbose)`, so I didn't want to get too far afield from the task or break a bunch of other things not related to ISO NE API by doing so. 

[I made a ticket for that change here](https://www.notion.so/gridstatus/Update-gridstatus-to-use-detailed-logger-11be835f42aa808f82c3d16d7365abd1?pvs=4). 

Could also make an argument for a larger change, going to a structured logging system like `structlog` but I'm fine without that tbh.